### PR TITLE
Add  bot5 2021/seminar-06

### PR DESCRIPTION
**Pull request by qhduan**

### CODE REVIEW

File: docs/_posts/2021-09-17-seminar-minutes-2021-06.md [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+    2. Vice-Chair: 张惟师
+
+## Attendees
+
+1. 王海良，运营 Chatopera 云服务，开发者友好的智能对话机器人平台 **本次分享者**
+1. 段清华，知未智能CTO&Co-founder **本次分享者**
+1. 李卓桓，Wechaty 作者, 基于 Conversational User Interface 的用户认证流程探讨
+1. 孙天祥，Authing Developer **本次分享者**
+1. 李佳芮, 句子互动创始人
+1. 董吉洋 **本次分享者**
```
Comment (lijiarui) [inline docs/_posts/2021-09-17-seminar-minutes-2021-06.md:30]: Please add introduction of jiyang:

真成投资 VP

File: docs/_posts/2021-09-17-seminar-minutes-2021-06.md [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+### 视频
+
+<div class="video-container" style="
+    position: relative;
+    padding-bottom:56.25%;
+    padding-top:30px;
+    height:0;
+    overflow:hidden;
+">
+<iframe width="560" height="315" src="https://www.youtube.com/embed/a8snyCoNAfQ" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (lijiarui) [inline docs/_posts/2021-09-17-seminar-minutes-2021-06.md:48]: Please add the issue link for the video description in youtube.

**Review by lijiarui — CHANGES_REQUESTED** [review]

File: docs/_posts/2021-09-17-seminar-minutes-2021-06.md (hunk 2 above)
Comment (qhduan) [reply docs/_posts/2021-09-17-seminar-minutes-2021-06.md:48]: added

File: docs/_posts/2021-09-17-seminar-minutes-2021-06.md (hunk 1 above)
Comment (qhduan) [reply docs/_posts/2021-09-17-seminar-minutes-2021-06.md]: added

**Pull request merged**